Libs/DICOM/Core/ctkDICOMDisplayedFieldGenerator.cpp

```diff
@@ -104,6 +104,7 @@ void ctkDICOMDisplayedFieldGenerator::updateDisplayedFieldsForInstance(
   QMap<QString, QString> &displayedFieldsForCurrentPatient )
 {
   Q_D(ctkDICOMDisplayedFieldGenerator);
+  Q_UNUSED(sopInstanceUID);
```

Comment:
@lassoan Is this warning fix legitimate or does it reveal an issue ?

Reply:
Good catch. Since cached tags are now fetched at higher level at once (to improve performance), SOP instance UID is not needed here anymore. It can be removed, and later, if for some reason is needed again then can still be added back.